Resources/Prototypes/DeltaV/Catalog/uplink_catalog.yml

```diff
@@ -45,4 +45,15 @@
   - !type:BuyerWhitelistCondition
     blacklist:
       components:
-      - SurplusBundle
\ No newline at end of file
+      - SurplusBundle
+      
+- type: listing
+  id: Paramedic Operative Reinforcement
+  name: uplink-reinforcement-radio-syndicate-paramedic-name
+  description: uplink-reinforcement-radio-syndicate-paramedic-desc
+  productEntity: ReinforcementRadioSyndicateParamedic
+  icon: { sprite: Objects/Devices/communication.rsi, state: old-radio }
+  cost:
+    Telecrystal: 10 # This Paramedic has taken an Oath to do no harm.
```

Comment:
The actual TC cost should probably be discussed a little more. Since the paramed is also bundled with a few syndicate items, he has a somewhat inflated inherent value. For example the cost of his NoSlips and Surgical Bundle should probably be added to this baseline cost.

Reply:
you can also just buy two combat medkits for that price which is more conveniant and i think has more healing, the advantage with the paramedic should be that, well, its another pair of hands. 

i think it should cost the same as the regular reinforcement, the inability to do harm is offset by their starting gear

Reply:
> you can also just buy two combat medkits for that price which is more conveniant and i think has more healing, the advantage with the paramedic should be that, well, its another pair of hands.
> 
> i think it should cost the same as the regular reinforcement, the inability to do harm is offset by their starting gear

I think the starting reinforcement cost is really dumb